modules/codegen/src/main/scala/com/twilio/guardrail/circe/CirceVersion.scala

```diff
@@ -13,7 +13,7 @@ object CirceVersion {
     def encoderObjectCompanion = q"ObjectEncoder"
     def print                  = q"pretty"
   }
-  case object V012 extends CirceVersion {
+  case object V013 extends CirceVersion {
```

Comment:
This should be left at V012, otherwise we'll be bumping this needlessly every minor version. The intent is really to separate the new source incompatibility from the old variant, Travis has otherwise been very good about keeping source compatibility so I don't expect we'll need a tremendous number of these. If another is created, we'll have to figure out the naming scheme at that point.

Thank you for working through this idea though!

Reply:
Resolved this so it can be merged and released